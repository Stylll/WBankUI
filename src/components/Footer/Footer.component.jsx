/* eslint-disable import/prefer-default-export */
import React from 'react';
import './Footer.styles.scss';

export const Footer = () => (
        <div className="footer">
            <p>Stephen Aribaba &copy; 2020</p>
        </div>
);
