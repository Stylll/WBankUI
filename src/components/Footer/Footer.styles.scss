@import '../../partials/colors';

.footer {
    background-color: $footer-bg-color;
    display: flex;
    justify-content: center;
    align-items: center;
    color: $white;
}
