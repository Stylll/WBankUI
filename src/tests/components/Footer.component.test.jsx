import { render } from '@testing-library/react';
import React from 'react';

import { Footer } from '../../components/Footer/Footer.component';

describe('Footer Component', () => {
  it('should render properly', () => {
    const { baseElement } = render(<Footer />);

    expect(baseElement).toMatchSnapshot();
  });
});
