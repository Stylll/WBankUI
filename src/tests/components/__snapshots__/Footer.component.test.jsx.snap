// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer Component should render properly 1`] = `
<body>
  <div>
    <div
      class="footer"
    >
      <p>
        Stephen Aribaba Â© 2020
      </p>
    </div>
  </div>
</body>
`;
